internal/cmd/egctl/config.go

```diff
@@ -113,10 +113,28 @@ func retrieveConfigDump(args []string, includeEds bool, configType envoyConfigTy
 // fetchRunningEnvoyPods gets the Pods, either based on the NamespacedName or the labelSelectors.
 // It further filters out only those Pods that are in "Running" state.
 // labelSelectors, if provided, take precedence over the pod NamespacedName.
-func fetchRunningEnvoyPods(c kube.CLIClient, nn types.NamespacedName, labelSelectors []string) ([]types.NamespacedName, error) {
+func fetchRunningEnvoyPods(c kube.CLIClient, nn types.NamespacedName, labelSelectors []string, allNamespaces bool) ([]types.NamespacedName, error) {
```

Comment:
IMO `nn` is not good, prefer `name` or `namespacedName`

Reply:
I did not change the previous vars, I think this is nit and not a blocker for this PR.